rotkehlchen/tests/db/test_db_upgrades.py

```diff
@@ -789,6 +793,17 @@ def test_upgrade_db_34_to_35(user_data_dir):  # pylint: disable=unused-argument
             cursor.execute(f'SELECT timestamp from {table_name}')
             assert cursor.fetchall() == expected_result
 
+    upgraded_tables = (
```

Comment:
Did you also edit the DB for the test?

Reply:
Do you mean `rotkehlchen/tests/data/v34_rotkehlchen.db`? 

I left a question about it at https://github.com/rotki/rotki/pull/4737#issuecomment-1227572615

Reply:
But yes, I added an entry to the web3_nodes table there if that's what you meant

Reply:
yes this is what I meant